product-service/handlers/getProductsById/getProductsById.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { withCorsHeaders } from "../../utils/withCorsHeaders";
+import { getProductsById } from ".";
+
+const mockProductList = [
+  {
+    count: 1,
+    description: "Description",
+    id: "1",
+    price: 10,
+    title: "Toy",
+    src: "url",
+  },
+];
+
+jest.requireActual("node-fetch");
+jest.mock("../../data/productList.json", () => [
+  {
+    count: 1,
+    description: "Description",
+    id: "1",
+    price: 10,
+    title: "Toy",
+    src: "url",
+  },
+]);
+
+describe("#getProductsById", () => {
+  it("should return product if it exists", async () => {
+    const targetProduct = mockProductList[0];
+    delete targetProduct.count;
+
+    const response = await getProductsById({
+      pathParameters: { id: targetProduct.id },
+    });
+
+    expect(response).toMatchObject(withCorsHeaders({ statusCode: 200 } as any));
```

Comment:
Why do you need to make it any?

Reply:
You mean why `as any` and not `as APIGatewayProxyResult`? If so, i reckon i just didn't notice it